### DISCUSSION TITLE
Repeatable fields in eNotices2 allow duplicate values to be saved.

### DISCUSSION THREAD

**Discussion by akashSrivastava-SAP** [OP-TED/eForms-SDK#268] [Q&A] [answered]
Hi,

Consider the repeatable field BT-531-Lot. It has 3 possible options - 'Works', 'Supplies' and 'Services'. In the eNotices2 UI, i can add these values any number of times. Is this behavior valid ? Please refer the screenshot below
![image](https://user-images.githubusercontent.com/119758441/206432496-8021d2de-3866-4810-88d7-d8f1420fa49b.png)

Also, if I post an XML containing these duplicated values to the CVS, i don't see any error for this.

![image](https://user-images.githubusercontent.com/119758441/206433009-c1b6e87f-69af-4ccd-80d0-357f4af50758.png)

Please provide some clarification here.

Thanks

**Comment by rouschr** [comment, marked as answer]
Hello,
note that eNotices2 is developed separately, for issues or questions concerning this project use this form:
https://enotices2.ted.europa.eu/help/contact

Maybe for the SDK side of things, @YvesJo has something to say about this?

**Reply by YvesJo** [reply]
A rule initially existed to prevent such situation at CVS level, it was however not ready for integration in TEDEN2. This rule will need to be set back at least at CVS level. At TEDEN2  level, it could be solved either way: use of check boxes or use of dynamic rule. Investigations and analysis are required to select the best option. Also, currently many dynamic rules are deactivated. The fix will be in a future release.
